src/app/components/media/ChangeMediaSource.js

```diff
@@ -43,80 +45,87 @@ function ChangeMediaSource({
   let createNew = null;
   if (can(projectMediaPermissions, 'create Source')) {
     createNew = (
-      <Button
-        style={{ color: 'var(--brandMain)' }}
+      <ButtonMain
+        variant="contained"
+        size="default"
+        theme="lightText"
         onClick={() => { createNewClick(input); }}
-        id="media-source__create-button"
-      >
-        <FormattedMessage
-          id="changeMediaSource.createSource"
-          defaultMessage="Create new"
-          description="allow user to create a new source"
-        />
-      </Button>
+        label={
+          <FormattedMessage
+            id="changeMediaSource.createSource"
+            defaultMessage="Create new"
+            description="allow user to create a new source"
+          />
+        }
+        buttonProps={{
+          id: 'media-source__create-button',
```

Comment:
Should this be an `id`? Could we end up with multiple instances of the same id on the page?

Reply:
idk 🤷 I just left it as an `id` because it was one before